packages/client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@knocklabs/client",
-  "version": "0.11.4",
+  "version": "0.12.0",
```

Comment:
I don't think you need to manually bump the version. That should be handled by changesets.